Localization/es-ES.hjson

```diff
@@ -1,3 +1,15 @@
+UI: {
+	Journey: "Viaje"
+	Normal: "Soy demasiado joven para minar"
+	Expert: "Ponme las pilas"
+	Master: "Soy la reencarnación de Cthulhu"
```

Comment:
(Same thing as I've said for #85 )

The names of the difficulties seem to be translated well, but I've been questioning for a while now whether or not they should be translated at all.. What do you think? I tried translating them to my first language, and either couldn't apply the puns (Expert), couldn't keep obviousness of the references (Master), or couldn't make the punified translation rhyme / be nice to pronounce (Normal.)

(Sorry for the late reply)

Reply:
Oooof sorry too for not responding in time either. I don't receive notifications from Github ;-;
Well, in Spanish I'd say the same. Almost all times translations lose their sense if they are literal, and, in many cases, I'm not sure of how I could translate them keeping their meaning. In this particular cases, I would say that Normal and Expert are not too bad here (they sound fine and there are similar puns in Spanish), but as you said the Master's one could be there problem.
Knowing that... I would keep them translated. About the Master's one... I suppose the reference is from a book in English or maybe some game dialogue (sorry I'm not sure where it comes from 🥲), so most Spanish people wouldn't recognise it in English probably. But if you prefer keeping them in English, it's fine.

Reply:
Alright, thanks.